### ISSUE TITLE
Problem of Best model

### ISSUE THREAD

**Issue by Lynkzhang** [juefeix/lbcnn.torch#5]
Since your Top1 and Top5 is acc rate not err rate, the condition of testTop1 < bestTop1 should be  testTop1 > bestTop1 isn't is right?

**Comment by juefeix** [comment]
Are you referring to line 58-61 in main.lua? I don't see an issue.

**Comment by Lynkzhang** [comment]
Yes, I think your code is forked from fb.resnet.torch right? And in trian.lua, I find that, fb's version gives Top5 and Top1 err rates, your version gives acc rate, so in main.lua, I guess you will fail to save the model which is actually the best one.
